Dockerfile.image-service

```diff
@@ -43,7 +43,8 @@ RUN mkdir $DATA_DIR && chmod 775 $DATA_DIR && chown $UID:$GID /data
 VOLUME $DATA_DIR
 ENV DATA_DIR=$DATA_DIR
 
-RUN dnf install -y nmstate && dnf clean all
+RUN dnf install -y epel-release && \
+    dnf install -y p7zip-plugins squashfs-tools && dnf clean all
```

Comment:
We need to add these in the downstream dockerfile as well (assuming theres a chance MCE doesn't move fully to konflux builds for 2.8). But this can be done after the merge.